### DISCUSSION TITLE
Notifiche unificate alexa

### DISCUSSION THREAD

**Discussion by Borghese** [indomus/forum#2778] [GENERALE - Discussioni generiche]
Ciao a tutti,

sto iniziando a impostare le prime automazioni su ha, e di conseguenza le prime notifiche
Son partito dopo aver installato uno shelly em per il monitoraggio della produzione del fotovoltaico, e dei prelievi/immissioni di energia in rete, oltre che a un paio di shelly1pm sui carichi pesanti di casa (questa parte ancora da espandere con ulteriori prese)

L'obiettivo è quello di ottimizzare al massimo l'autoconsumo, cioè uso immediato dell'energia che autoproduco, che mi conviene appunto consumare piuttosto che immetterla in rete, e magari prelevarla un momento successivo, visto che la pago molto piu di quello che la pagano a me. 

La prima automazione che ho creato, è quindi per avviare la produzione di acqua calda sanitaria, tramite boiler a pdc, (sulla presa del quale ho installato uno shelly1pm) in modo da attivare lo switch solo se sto immettendo in rete almeno -500watt (la mia casa in condizioni "dormienti" assorbe tra i 2 e i 300watt, e il boiler pdc assorbe a regime ca 500watt, per cui se il disavanzo tra quello che produco, e quello che consumo, è uguale o superiore a -500watt, vorrei attivare lo scaldabagno)

Unica cosa che non riesco a fare, è gestire le notifiche tramite gli alexa che ho in casa, mi spiego meglio  : 
Ho in 3 stanze diverse, rispettivamente un echo pop, e due echo dot
Li vedo regolarmente in home assistant, ma vorrei creare un unica notifica simultanea su tutti e tre gli alexa, ma non ci riesco, posso solo creare tre notifiche separate, una per ogni alexa. 

Io pensavo di usare l'entità "in tutta la casa" che crea l'integrazione di alexa, ma anche se simulando l'automazione non rilevo errori, queste non parlano (ovviamente singolarmente nessun problema)

Ho provato anche a usare l'entità "alexa_media" ma questa mi da un unkown error

Ho anche provato a creare un aiutate, creando un gruppo mettendoci le tre alexa dentro, ma poi questo gruppo non me lo ritrovo tra i sistemi di notifica dell'automazione

Mi sapete aiutare ?

**Comment by Scialla** [comment]
se mandi sui singoli echo usi:
```
- service: notify.alexa_media
  data:
    target: media_player.echo_tuoecho
    data:
      type: tts
      method: spoken
    message: "Tuo Messaggio"
```
se mandi sul gruppo devi usare come `type: announce`

fai già così?

**Reply by Borghese** [reply]
Allora, l'ho fatta così  : 

```python
  - service: notify.alexa_media
    data:
      message: Prova
      target: media_player.gruppo_alexa
      data:
        type: announce
        method: spoken
  mode: single
``` 

ma nulla da fare, non funziona....

Poi ho modificato il target, non usando il gruppo, ma il servizio "in tutta la casa" e così funziona...

```python
  - service: notify.alexa_media
    data:
      message: Prova
      target: media_player.in_tutta_la_casa
      data:
        type: announce
        method: spoken
  mode: single
``` 

non capisco perchè... a sto punto posso anche eliminare il gruppo allora...
